mage_ai/server/server.py

```diff
@@ -215,12 +220,13 @@ async def main(
         # We need to sleep for a few seconds after creating all the tables or else there
         # may be an error trying to create users.
         sleep(3)
-        user = User.query.filter(User.owner == True).first()  # noqa: E712
+        Role.create_default_roles()
+        user = User.query.filter(User._owner == True).first()  # noqa: E712
         if not user:
             print('User with owner permission doesn’t exist, creating owner user.')
             if AUTHENTICATION_MODE.lower() == 'ldap':
                 user = User.create(
-                    owner=True,
+                    roles_new=[Role.query.filter(Role.name == 'owner').first()],
```

Comment:
I would suggest you adding a method to the Role class to get the predefined roles and wrap the method with `@safe_db_query` decorator.